<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectSets</class>
 <widget class="QWidget" name="ObjectSets">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Наименование:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="txtName">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdSave">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdDrop">
          <property name="text">
           <string>Сбросить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Соединение:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="txtConnection">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Изменить...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Строка связи:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtConnString">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Попыток связи:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="txtRecall">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Действия:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="cmdAddFeeder">
            <property name="text">
             <string>Добавить фидер</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdRemove">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtConnection</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>txtConnString</tabstop>
  <tabstop>txtRecall</tabstop>
  <tabstop>cmdRemove</tabstop>
  <tabstop>cmdDrop</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cmdSave</sender>
   <signal>clicked()</signal>
   <receiver>ObjectSets</receiver>
   <slot>saveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>773</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdDrop</sender>
   <signal>clicked()</signal>
   <receiver>ObjectSets</receiver>
   <slot>dropChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>789</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdRemove</sender>
   <signal>clicked()</signal>
   <receiver>ObjectSets</receiver>
   <slot>removeItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdAddFeeder</sender>
   <signal>clicked()</signal>
   <receiver>ObjectSets</receiver>
   <slot>appendChild()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>713</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ObjectSets</receiver>
   <slot>connectionChangeRequest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>754</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dropChanges()</slot>
  <slot>saveChanges()</slot>
  <slot>removeItem()</slot>
  <slot>appendChild()</slot>
  <slot>connectionChangeRequest()</slot>
 </slots>
</ui>
